addon/components/rdf-input-fields/concept-scheme-selector.js

```diff
@@ -32,21 +33,28 @@ export default class RdfInputFieldsConceptSchemeSelectorComponent extends InputF
 
   loadOptions() {
     const metaGraph = this.args.graphs.metaGraph;
-    const fieldOptions = this.args.field.options;
+    const conceptSchemeUri = this.findFieldOption(
+      'conceptScheme',
+      EXT('conceptScheme')
+    );
+    const isSearchEnabled = this.findFieldOption(
+      'searchEnabled',
+      EXT('searchEnabled'),
+      'boolean'
+    );
```

Comment:
Should we make a findAllFieldOptions method that will have an array of property/predicate values?

Reply:
As how it is now I don't mind it having it separatly
```js
    const conceptScheme = this.findFieldOption('conceptScheme', 'node');
    const isSearchEnabled = this.findFieldOption('searchEnabled', 'boolean');
```